README.md

```diff
@@ -1 +1,45 @@
-# unicorn-sokoban
\ No newline at end of file
+# unicorn-sokoban
+Sokoban features a warehouse that is depicted as a grid of squares, each one representing either a floor section or a wall section. Some floor squares contain boxes and some are marked as storage locations. The player, often represented as a worker character, can move one square at a time horizontally or vertically onto empty floor squares, but cannot pass through walls or boxes. The number of boxes equals the number of storage locations. The puzzle is solved when all boxes are located on storage locations.
+
+
+Inspiration
+Your assignment is to create you version of this classical game. For inspiration, check out this youtube video. There exists more videos on yotube. Just google for ut.
+
+You can also try the orignal game here. There are countless other versions of the games as well.
+
+Group Assignment - Scrum
+This is a group assignment which means you need to work together in your team. The requirement here is that you work according to scrum. Now we have talked about scrum earlier in this course. Now I don't expect you to do it flawlessly but you will try, and see this as good practise for the future. When working with scrum you will need to make use of the five scrum events that you have learned.
```

Comment:
Now I don't expect you to do it flawlessly but you will try, and see this as good practise for the future. When working with scrum you will need to make use of the five scrum events that you have learned.
===
first-person appeal

Reply:
I suggest adding to the description of the team members, as well as a link to our scrum board, which rule we introduced before the merge process:
![Screenshot 2024-03-20 at 11 08 10](https://github.com/SofiaHjerpe/unicorn-sokoban/assets/36052921/4906e126-cc23-434c-9a6c-fa6e8cc24194)
===
what methods did we use to solve the problems in the first sprint, fix the intermediate result, and later add to the description how far we have progressed